pxtsim/runtime.ts

```diff
@@ -223,7 +223,7 @@ namespace pxsim {
                     aws.forEach(aw => aw());
                 }
             }
-            if (!this.handler || this.events.length > this.max) return;
+            if (this.handlers == [] || this.events.length > this.max) return;
```

Comment:
should be `this.handlers.length == 0`; `==` is probably ref-equality

Reply:
thanks.